src/reducers/articles-update.reducers.js

```diff
@@ -0,0 +1,27 @@
+const initialState = {
+  fetching: false,
+  fetched: false,
+  articles: [],
+  error: null,
```

Comment:
@walsamlee we have already general exception handler function, which you already used.so we don't need to have error in this state anymore since it's not being used.

Reply:
Thanks for the feedback, this has been corrected.